### PR TITLE
fix logo animation in main nav

### LINKED ISSUE #226: [Main Navigation] Possible animation error shows duplicate logo
image

### PR DESCRIPTION
**Pull request by wadie**
fixes #226

### CODE REVIEW

File: _includes/nav.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <nav class="navbar navbar-expand-md navbar-light fixed-top">
     <div class="container">
         <a class="navbar-brand" href="/">
```
Comment (ghislaineguerin) [inline _includes/nav.html:5]: @wadie I worry that without CSS we don't have any reference to our logo or brand here. Is it possible to implement this differently?

**Comment by ghislaineguerin** [comment]
@wadie I've added a comment on the code. I think we need to still see our brand without CSS, just not sure how that conflicts with the animation.

File: _includes/nav.html (hunk 1 above)
Comment (wadie) [reply _includes/nav.html]: I'll make sure we use normal `` tags in order to have `alt` for the logo. 👍

**Comment by ghislaineguerin** [comment]
👍 looks great

**Pull request merged**